terraform/main.tf

```diff
@@ -0,0 +1,175 @@
+# Terraform configuration for AWS Batch job resources that can run the model.
```

Comment:
I tried to be extra thorough in documenting this module, to help less experienced readers understand what all of these entities do. Let me know if you think it's too much or too little!

Reply:
**praise:** This is great! Thanks for all the extra docs. Useful to me as a relative terraform noob.